jumpcloud-ADMU/Powershell/Private/DisplayForms/Form.ps1

```diff
@@ -274,7 +274,52 @@ Function Show-SelectionForm {
     $img_localAccountValid.Source = Get-ImageFromB64 -ImageBase64 $ErrorBase64
     $img_localAccountPasswordInfo.Source = Get-ImageFromB64 -ImageBase64 $BlueBase64
     $img_localAccountPasswordValid.Source = Get-ImageFromB64 -ImageBase64 $ActiveBase64
-    # Define misc static variables
+
+    $loadingForm = New-Object System.Windows.Forms.Form
+    $loadingForm.Text = "Loading"
+    $loadingForm.StartPosition = 'CenterScreen'
+    $loadingForm.FormBorderStyle = 'None'
+    $loadingForm.BackColor = [System.Drawing.Color]::FromArgb(9, 26, 56)
+    $loadingForm.ClientSize = New-Object System.Drawing.Size(840, 320)
+    $loadingForm.TopMost = $true
+    $loadingForm.ShowInTaskbar = $false
+
+    # ---- Title ----
+    $title = New-Object System.Windows.Forms.Label
+    $title.Dock = 'Top'
+    $title.Height = 150
+    $title.TextAlign = 'MiddleCenter'
+    $title.Text = "JumpCloud ADMU"
+    $title.ForeColor = [System.Drawing.Color]::White
+    $title.Font = New-Object System.Drawing.Font("Segoe UI", 24, [System.Drawing.FontStyle]::Bold)
+
+    # ---- Message ----
+    $msg = New-Object System.Windows.Forms.Label
+    $msg.Dock = 'Bottom'
+    $msg.Height = 145
+    $msg.TextAlign = 'MiddleCenter'
+    $msg.Text = "Please wait while gathering system information..."
+    $msg.ForeColor = [System.Drawing.Color]::FromArgb(220, 230, 255)
```

Comment:
Okay my only suggestion here should be to make sure we are picking something from our primary color palette. I'll send you a document to reference here.